backend/inspirehep/search/factories/search.py

```diff
@@ -18,16 +18,10 @@
 
 
 def can_request_fields():
-    allowed_mimetypes = [
-        "application/json",
-        "application/vnd+inspire.record.ui+json",
-        "application/vnd+inspire.record.raw+json",
-    ]
     has_accept_mimetypes = len(request.accept_mimetypes) > 0
-    if (
-        has_accept_mimetypes
-        and next(request.accept_mimetypes.values()) not in allowed_mimetypes
-    ):
+    if has_accept_mimetypes and next(
+        request.accept_mimetypes.values()
+    ) in current_app.config.get("FORBIDDEN_MIMETYPES_FOR_API_FILTERING"):
```

Comment:
this is not going to work if the key is not present, as it will be `... in None`. So either pass a default arg to `get`, or drop the `get` and use `[...]`.